application/views/header.php

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en" class="h-100">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="<?php echo base_url('assets/css/bootstrap.min.css'); ?>">
+    <link rel="stylesheet" href="<?php echo base_url('assets/css/style.css'); ?>">
+
+	<script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
We should always put the js file below the body, not on the head, this could decrease the page loading speed
If you want the script to be loaded immediately after the HTML loaded then use `defer attribute`

Reply:
I put it after <main> closing tag.Do you think it's ok?